packages-next/admin-ui/src/next-config.ts

```diff
@@ -1,6 +1,6 @@
 // @ts-ignore
-import withPreconstruct from '@preconstruct/next';
 import Path from 'path';
+import withPreconstruct from '@preconstruct/next';
```

Comment:
Can't add a suggestion due to deleted line, @timleslie  the import was moved but not the ts-ignore